WordPress/src/main/res/layout/quick_start_completed_tasks_list_header.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <View
+        android:id="@+id/top_spacing"
+        android:layout_width="wrap_content"
+        android:layout_height="@dimen/margin_extra_large"
+        android:background="@color/grey_lighten_30"/>
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="48dp"
+        android:background="?attr/selectableItemBackground"
+        android:paddingEnd="@dimen/margin_extra_large"
+        android:paddingStart="@dimen/margin_extra_large">
+
+        <TextView
+            android:id="@+id/complete_tasks_header_label"
+            android:layout_width="wrap_content"
```

Comment:
```suggestion
            android:ellipsize="end"
            android:layout_width="wrap_content"
```

We should add the `android:ellipsize="end"` attribute to avoid translations being clipped by the chevron image.